src/common/entities/todo.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, Enum, PrimaryKey, Property } from '@mikro-orm/core';
+import { v4 as uuidv4 } from 'uuid';
+import { ETodoPriority } from '../enums/todo-priority.enum';
+import { TodoRepository } from '../../todos/todos.repository';
+
+@Entity({ repository: () => TodoRepository })
+export class Todo {
+  @PrimaryKey({ type: 'uuid' })
+  id: string = uuidv4();
+
+  @Property()
+  title!: string;
+
+  @Property({ type: 'text' })
+  description!: string;
+
+  @Property({ default: false })
+  isCompleted: boolean = false;
```

Comment:
```suggestion
  @Property({ default: false, fieldName: "is_completed" })
  isCompleted: boolean = false;
```

We should add fieldName (in snake_case) explicitly for all of the attributes.